### DISCUSSION TITLE
All checks are failed

### DISCUSSION THREAD

**Discussion by eliasrivarola99** [vercel/community#4348] [Account Issue] [answered]
### Question

Hi, I upload the repository, but when I want to open it with Vercel doesn't let me, I click more details and looks like it's the first time that open up my Vercel's account. What can I do?

**Comment by swarnava** [comment]
Hi, can you share a screencast?

**Reply by eliasrivarola99** [reply]
![Screenshot 2023-10-03 at 14 27 55](https://github.com/vercel/community/assets/139979615/ac9d104d-ef45-4d93-b012-b129f7b0df59)
That's what I see, when I click on details, the page goes to Vercel, to sign in or log in, I log in, but it looks likes its the first time and says welcome to the team and all that things

**Reply by swarnava** [reply, marked as answer]
Thanks! Could you confirm if these settings are intact: https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel ?

**Reply by eliasrivarola99** [reply]
Can be a problem that my username on GitHub its different that the one on Vercel? because my coworker have the same one in both platforms

**Reply by swarnava** [reply]
The GitHub username shouldn't be a problem as long as your Vercel email and GitHub email are same.
